### PR TITLE
swig 3.0.8

### PR DESCRIPTION
**Pull request by bartleusink**
Hi!

This PR upgrades swig from 3.0.7 to 3.0.8

### CODE REVIEW

**Comment by mikemcquaid** [comment]
Test is failing on 10.9: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/39343/version=mavericks/testReport/junit/brew-test-bot/mavericks/test_swig/

**Comment by bartleusink** [comment]
The problem is that the test is compiled against the system ruby headers (ruby 2.0) while the test is executed with ruby 1.8.7.
Basically, "-I/System/Library/Frameworks/Ruby.framework/Headers/" should be changed to something like "/usr/local/opt/ruby187/lib/ruby/1.8/i686-darwin15.2.0", depending on the ruby version used in the test.

What's the best way to detect the ruby version from within brew test?

File: Library/Formula/swig.rb [hunk 1]
```diff
@@ -40,9 +40,14 @@ def install
       require "./test"
       puts Test.add(1, 1)
     EOS
+    headers = if ENV['HOMEBREW_RUBY'] == "1.8.7"
+      `ruby -rrbconfig -e 'print RbConfig::CONFIG["archdir"]'`
```
Comment (bfontaine) [inline Library/Formula/swig.rb]: Would this code work with other Rubies as well (e.g. 2.x)?

**Comment by UniqMartin** [comment]
Can you remove the modifications that deal with `HOMEBREW_RUBY` and give it another try? We very recently changed the way we test with Ruby 1.8 on Mavericks, so the formula should again see the system Ruby (instead of the one used to run Homebrew). Thanks!

File: Library/Formula/swig.rb (hunk 1 above)
Comment (bartleusink) [inline Library/Formula/swig.rb]: Unfortunately not,

The ruby that is by default installed on OSX has its headers installed in `"/System/Library/Frameworks/Ruby.framework/Headers/"`
Ruby <= 1.8 headers are installed in `RbConfig::CONFIG["archdir"]`
Ruby >= 1.9 headers are installed in `RbConfig::CONFIG["rubyhdrdir"]`

**Comment by bartleusink** [comment]
Looks like it works without `HOMEBREW_RUBY` trickery now!

**Comment by bfontaine** [comment]
Thank you for your contribution @bartleusink; we appreciate it! :tada:

**Pull request closed without merging**